packages/trie/src/trie/trie.ts

```diff
@@ -35,38 +35,39 @@ interface Path {
  * The API for the base and the secure interface are about the same.
  */
 export class Trie {
+  #opts: TrieOptsWithDefaults = {
```

Comment:
What is this `#` standing for respectively doing? 🤔

Reply:
Ah, just googled a bit, this is an ES2022 property which is beyond our build target, let's please not introduce any new language constructs here last minute.

This property is super useful, but not anything any more for this round of breaking releases. I can very well imagine a broad set of refactoring around this along the next breaking release round. 🙂